CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 ### Feature
 
+- Sentry integration
+- All the environment variables defined at runtime, that have the "RAZZLE_" prefix, are now available in the browser under window.env
```

Comment:
@zotya where does this changelog entry come from?

Reply:
@tisto 
The functionality was implemented in https://github.com/plone/volto/commit/00c93ca60f2ea4a8987b8c5d6358efde3f541992
with changes in the files:
src/runtime_config.js
src/server.jsx

I've added a small documentation about it in:
https://github.com/plone/volto/commit/5237775cbe56598003a29f5ba32cfe8c606079f6